src/app/backend/integration/handler.go

```diff
@@ -0,0 +1,37 @@
+package integration
+
+import (
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"github.com/kubernetes/dashboard/src/app/backend/integration/api"
+)
+
+// IntegrationHandler manages all endpoints related to integrated applications, such as state.
+type IntegrationHandler struct {
+	manager IntegrationManager
+}
+
+// Install creates new endpoints for integrations.
```

Comment:
An addon most likely needs to know the namespace selection. Would this be somthing to add?